hathor/manager.py

```diff
@@ -1077,7 +1077,7 @@ def _log_feature_states(self, vertex: BaseTransaction) -> None:
             features_states=state_by_feature
         )
 
-        features = [Feature.NOP_FEATURE_4, Feature.NOP_FEATURE_5, Feature.NOP_FEATURE_6]
+        features = [Feature.NOP_FEATURE_1, Feature.NOP_FEATURE_2]
```

Comment:
Why this change?

Reply:
This is simply limiting which features are logged when they're `ACTIVE`, for the phased testing. Before, it was selecting the features used by the testnet phased testing (4, 5, 6). Now that it's complete, I'm selecting only the ones that will be used by mainnet (1, 2). After the whole process is complete, all this code is going to be removed.

Reply:
Got it. So it's a fix since the mainnet features were supposed to already be in the list, right?

Reply:
Yes, I didn't call it fix because the mainnet process hasn't started yet, but I guess it could be

Reply:
Got it. Thanks!